docs/documentation/documentation_tips.md

```diff
@@ -43,15 +43,20 @@ For example: `By default, the integration will import PagerDuty incidents data a
 
 ## Common Integration Parameters
 The most commonly used integration parameters:
-
-| Parameter name | Display name |
-|---|---|
-| API token/key | API Token/ API Key/ API Secret. This parameter should match the product. |
-| URL | Server URL |
-| insecure | Trust any certificate (not secure) |
-| proxy |  Use system proxy settings |
-| Threshold | The minimum number/severity/score ... |
-| Limit | The maximum number of... |
+| Parameter name | Display name | Notes |
+|---|---|---|
+| API token/key | API Token/ API Key/ API Secret. |Provided by the third party integration. | |
+| URL | Server URL | |
+| insecure | Trust any certificate (not secure) | When ‘trust any certificate’ is selected, the integration ignores TLS/SSL certificate validation errors. Used to test connection issues or connect to a server without a valid certificate. | |
+| proxy |  Use system proxy settings |Runs the integration instance using the proxy server (HTTP or HTTPS) that you defined in the server configuration.|
+| Threshold | The minimum number/severity/score ... | |
+| Limit | The maximum number of... |   |         
+
+## Parameters found in all integrations:
+| Parameter | Notes 
+|---|---| 
+| Run on Single engine / Run on  Load Balancing Group | Communications between the XSOAR server and the 3rd party service are executed through the selected engine or load balancing group, not directly.|
+| Do not use by default | <p>Use to avoid exceeding API quotas</p> <ul><li> <p>When enabled, commands from this integration will not be available through the CLI, when you run a generic command that uses all available integration commands.</p></li> <li> <p>To use a command at the CLI from an instance with ‘do not use by default’ enabled, you need to specify the instance with the ‘using’ argument. </p></li><li> <p>Commands from this integration will still run in Playbooks, unless you have also added the ignore.default.in.playbooks server configuration, set to true. </p><li> If you have multiple instances of this integration, and the ‘do not use by default’ option is selected in one or more of the instances, the setting will apply for all instances of the integration. |
```

Comment:
Some HTML tags used here are not supported in xsoar.pan.dev. We use MDX format. For bullet lists, please use the [markdown format](https://wordpress.com/support/markdown-quick-reference/).

Reply:
@fvigo  - Is there a way to include a bulleted list within a table, without these html tags? If not, will figure out something else.

Reply:
@tschanfeld no, if it's in a table you are right to use the html tags, as in this example: https://github.com/demisto/content-docs/blob/master/docs/contributing/checklist.md

You probably shouldn't use <p> though.

In your case the build is failing because at least an </li> is not closed.